lib/threatstack/version.rb

```diff
@@ -1,3 +1,3 @@
 module Threatstack
-  VERSION = "0.2.0"
```

Comment:
Are you confident this should be the first "stable" release? Are you publishing this gem to rubygems?

Reply:
I'm confident that it doesn't support earlier versions so it's a major upgrade, not a minor. I'm fairly confident that there are no major bugs since I wrote some pretty comprehensive unit tests and manually tested all endpoints. But I'm sure there will be some.

Reply:
And yes I am publishing this to rubygems